yeat/config/config.py

```diff
@@ -17,20 +17,20 @@
 
 
 class AssemblyConfig:
-    def __init__(self, config, threads):
+    def __init__(self, config, threads=1, bandage=False):
         self.config = config
-        self.validate()
+        self.validate_config_keys()
         self.threads = threads
-        self.samples = {}
-        self.assemblies = {}
-        self.create_sample_and_assembly_objects()
-        self.validate_samples_to_assembly_modes()
+        self.bandage = bandage
+        self.samples = self.create_sample_objects()
+        self.assemblies = self.create_assembly_objects()
+        self.expected_files = self.get_expected_files()
```

Comment:
`self.expected_files` will have all the collated expected files from both sample and assembly objects.